src/ingameWindows/iwBuilding.cpp

```diff
@@ -100,6 +103,32 @@ iwBuilding::iwBuilding(GameWorldView& gwv, GameCommandFactory& gcFactory, nobUsu
 
     AddText(10, DrawPoint(113, 50), _("(House unoccupied)"), COLOR_RED, FontStyle::CENTER, NormalFont);
 
+    if(gwv.GetWorld().GetGGS().isEnabled(AddonId::SHOWRESOURCES))
+    {
+		if(BuildingProperties::IsMine(building->GetBuildingType()))
+		{
+			Resource::Type res;
+			switch(building->GetBuildingType())
+			{
+				case BLD_GOLDMINE: res = Resource::Type::Gold; break;
```

Comment:
This `Resource::Type::Gold` class/namespace mixing is a Visual Studio only addon to the c++0x stuff, while the project also builds for older versions. so this is breaking the auto build.